pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java

```diff
@@ -146,9 +146,7 @@ public static void main(String[] args) throws Exception {
     private static void deleteLedger(BookKeeper bookKeeper, long ledgerId) {
         try {
             bookKeeper.deleteLedger(ledgerId);
-            if (log.isDebugEnabled()) {
-                log.debug("Delete ledger id: {}", ledgerId);
-            }
+            log.debug("Delete ledger id: {}", ledgerId);
```

Comment:
This is needed, otherwise the `ledgerId` which is a `long`, gets converted into a `Long` causing a heap allocation.

Reply:
Thank you for your review and suggestions. I will make some changes.